sim_pipeline/gg_lens.py

```diff
@@ -5,6 +5,34 @@
 from lenstronomy.LensModel.Solver.lens_equation_solver import LensEquationSolver
 
 
+def image_separation_from_positions(image_positions):
+    """
+    calculate image separation in arc-seconds
+
+    :param image_positions: list of image positions in arc-seconds
```

Comment:
can you be more specific? in the code, you call ten(image_positions[0]) ==2: meaning the input is different than what you stated.